apps/web/pages/api/teams/googleworkspace/add.ts

```diff
@@ -0,0 +1,35 @@
+import { google } from "googleapis";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import getAppKeysFromSlug from "@calcom/app-store/_utils/getAppKeysFromSlug";
+import { WEBAPP_URL } from "@calcom/lib/constants";
+
+const scopes = ["https://www.googleapis.com/auth/admin.directory.user.readonly"];
+
+let client_id = "";
+let client_secret = "";
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (req.method === "GET") {
+    // Get appKeys from google-calendar
+    const appKeys = await getAppKeysFromSlug("google-calendar");
+    if (typeof appKeys.client_id === "string") client_id = appKeys.client_id;
+    if (typeof appKeys.client_secret === "string") client_secret = appKeys.client_secret;
```

Comment:
```suggestion
export default async function handler(req: NextApiRequest, res: NextApiResponse) {
  if (req.method === "GET") {
    // Get appKeys from google-calendar
    const { client_id, client_secret } = await getAppKeysFromSlug("google-calendar");
```

Reply:
This code example is deprecated we can just check from DB and error if there aren't any.

Reply:
Are we sure? these still return prisma JSON objects and we would need to still check if they were strings -> throw if they dont exist 
<img width="225" alt="CleanShot 2023-05-19 at 09 20 07@2x" src="https://github.com/calcom/cal.com/assets/55134778/3edbaad6-b63d-493e-8f4f-1561bb43cfb6">

Reply:
The check stays, what doesn't stay is the module-scoped variables